azext_edge/edge/providers/orchestration/template.py

```diff
@@ -816,7 +816,7 @@ def copy(self) -> "TemplateBlueprint":
 def get_insecure_listener(instance_name: str, broker_name: str) -> dict:
     return {
         "type": "Microsoft.IoTOperations/instances/brokers/listeners",
-        "apiVersion": "2024-08-15-preview",
+        "apiVersion": "2024-09-15-preview",
```

Comment:
should prob be a constant somewhere but thats a later issue

Reply:
This is true, it's a constant in tests but really should move up into a common module